admin_ui/backend/api/system.py

```
@@ -4619,56 +4619,78 @@ def _resolve_app_name() -> str:
     try:
-        if os.path.exists(manifest_path):
-            with open(manifest_path, "r") as f:
-                manifest = _json.load(f)
+        async with httpx.AsyncClient(timeout=timeout, transport=transport) as client:
+            for url in candidates:
+                try:
+                    resp = await client.get(url)
+                    if resp.status_code == 200:
+                        data = resp.json()
+                        return bool(data.get("ari_connected"))
+                except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)